ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -528,7 +531,7 @@ def setUp(self):
         #
         self.generate_from_t1()
         self.generate_from_vlan()
-        self.generate_ping_dut_vlan_intf()
+        self.generate_ping_dut_lo()
```

Comment:
Why do you need this change?

Reply:
I change the behavior from ping vlan interface IP to lo ip, so it will work for a topo without any vlan such as T1. Based on experiment, I found the behaviors are roughly same.

---
In reply to: [244617487](https://github.com/Azure/sonic-mgmt/pull/766#discussion_r244617487) [](ancestors = 244617487)